README_zh_CN.md

```diff
@@ -356,4 +356,4 @@ You can use these commands to get more information about the experiment
 
 ## **许可协议**
 
-代码库遵循 [MIT 许可协议](LICENSE)
\ No newline at end of file
+代码库遵循 [MIT 许可协议](LICENSE)
```

Comment:
please revert the changes to chinese doc

Reply:
I didn't change this file in fact. I believe there might be some invisible characters modified by the editor. I will try to figure it out.

Reply:
updated